ci/Dockerfile.dockerhub

```diff
@@ -0,0 +1,46 @@
+# This Dockerfile is used to create images published to DockerHub. It needs to
+# be kept in sync with the corresponding Dockerfile we use and test in CI
+# (docker/Dockerfile.ubuntu-20).
+
+FROM ubuntu:focal
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ARG ZEEK_LTS=1
+ARG ZEEK_VERSION=3.0.11-0
+
+CMD ["sh"]
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends curl ca-certificates gnupg2 \
+ # Zeek.
+ && mkdir -p /tmp/zeek-packages \
+ && cd /tmp/zeek-packages \
+ && if [ -n "${ZEEK_LTS}" ]; then ZEEK_LTS="-lts"; fi && export ZEEK_LTS \
+ && curl -L --remote-name-all \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeek${ZEEK_LTS}_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeek${ZEEK_LTS}-core_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeekctl${ZEEK_LTS}_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeek${ZEEK_LTS}-core-dev_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/libbroker${ZEEK_LTS}-dev_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeek${ZEEK_LTS}-libcaf-dev_${ZEEK_VERSION}_amd64.deb \
+ && apt-get install -y libpcap0.8 libpcap-dev python libssl-dev zlib1g-dev libmaxminddb0 libmaxminddb-dev \
+ && dpkg -i *.deb \
+ && cd - \
+ && rm -rf /tmp/zeek-packages \
+ # Spicy build and test dependencies.
+ && apt-get install -y --no-install-recommends git ninja-build ccache bison flex g++ libfl-dev python3 python3-pip zlib1g-dev jq locales-all python3-setuptools python3-wheel make \
+ && pip3 install "btest>=0.66" pre-commit \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* \
+ # Install a recent CMake.
+ && mkdir -p /opt/cmake \
+ && curl -L https://github.com/Kitware/CMake/releases/download/v3.18.0/cmake-3.18.0-Linux-x86_64.tar.gz | tar xzvf - -C /opt/cmake --strip-components 1
+
+ENV PATH="/opt/zeek/bin:/opt/cmake/bin:${PATH}"
+
+# Install Spicy.
+ADD . /opt/spicy/src
+RUN cd /opt/spicy/src && ./configure --generator=Ninja --prefix=/opt/spicy --with-zeek=/opt/zeek && ninja -C build install && rm -rf build
```

Comment:
Are we going to keep installing the Zeek plugin and analyzers through this as well? If so, I'd add an explicit `--build-zeek-plugin=yes` so that it won't break when we switch the default. And currently it won't install the analyzers.

Or we go the zkg route here as well, either is way fine I think.

Reply:
I went with Spicy w/o plugin and installation of additional Zeek components via zkg.